### DISCUSSION TITLE
rtorrent 0.15.0/0.15.0

### DISCUSSION THREAD

**Discussion by liquid187** [swizzin/swizzin#1205] [Feature Requests] [answered]
hi. could you please adapt the swizzin script so that you can upgrade to rtorrent 0.15.0/0.15.0? It fixes some of the bugs I was having problems with.

**Comment by Seroczynski** [comment]
During a fresh install the installer mentions the REPO version is (still) 0.9.8. I wonder why that is.

Please update this.

**Reply by liaralabs** [reply]
Because it is

https://packages.debian.org/bookworm/rtorrent
https://packages.ubuntu.com/noble-updates/rtorrent

**Comment by flying-sausages** [comment, marked as answer]
Released to master, there's a couple options available

**Comment by MorphyDK** [comment]
Hi 

Just some noob questions , and im sorry for asking , but since I don't know better, I thought this maybe was the right place to ask  :) .

First of all , im a HUGE fan of this Swizzin box setup - Simple and so good and very stable . LOVE IT

I saw an upgrade command : [https://swizzin.ltd/applications/rtorrent#how-to-upgradedowngrade](url)

When I ran the command - I saw a version 0.15.5 (UDNS)  .. .UDNS ? What is that :) ?

Also the update system says  "All versions other than repo will be locally compiled from source" , when I scroll down the only repo I see is (0.9.8) . Should I be aware of something? 

I tried to run the updater for 0.15.5 (UDNS) and my system was updated perfectly . But I don't know what this UDNS is :)

Last question . 

[https://github.com/rakshasa/rtorrent/releases](url) shows that there is a 0.16.8 version available . And here is my noob question , how come the updater don't show this when I run "sudo box upgrade rtorrent" ? 

Thanks for all the great work. 

Kr

Morphy
